packages/generator-langium/src/index.ts

```diff
@@ -8,9 +8,10 @@ import Generator from 'yeoman-generator';
 import _ from 'lodash';
 import chalk from 'chalk';
 import path from 'path';
+import { cwd } from 'node:process';
```

Comment:
This import doesn't work from what I can tell. Seems the package.json is missing a dev dep of `@types/node` to get this to work.